jsonalchemy/validator.py

```diff
@@ -113,9 +115,23 @@ def _validate(self, document, schema=None, update=False):
                 if not self.allow_unknown:
                     self._error(field, errors.ERROR_UNKNOWN_FIELD)
 
+        # Remove special rules, as they can cause error in Cerberus validation.
```

Comment:
I think we should not do that, this way we are not applying this validation rules.
Actually the `_validate` method could be removed see https://github.com/inveniosoftware/jsonalchemy/pull/13

Maybe is worth to check what Cerberus gets from us and why it fails.
I think we shouldn't do this, we should instead check why Cerberus crashes